code/modules/atmospherics/components/unary/outlet_injector.dm

```diff
@@ -158,10 +198,9 @@
 	update_underlays()
 
 /obj/machinery/atmospherics/unary/outlet_injector/attack_hand(mob/user as mob)
-	if (!src.allowed(user)) // ID check, to prevent randos from switching off secure atmos equipment.
```

Comment:
yes, i removed the access check
if you don't want people switching off your "secure atmos equipment", secure it better.